certified-connectors/Tilkee-V3/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2241 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.3",
+    "description": "Tilkee is an online platform that makes cross-functional collaboration happen. Combining deep knowledge of sales enablement with augmented intelligence, the vision of the team behind Tilkee is to align marketing with sales and guide Sales, Product Marketing, Channel Enablement, and Customer Success teams towards digital transformation.",
+    "title": "Tilkee V3",
+    "contact": {
+      "name": "Tilkee support",
+      "url": "https://help.tilkee.com",
+      "email": "support@tilkee.com"
+    }
+  },
+  "host": "endpoints.tilkee.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v3/oauth/authorize/": {},
```

Comment:
@FloxoS Empty operation definitions can be removed as they would not add any value to your connector.